### PR TITLE
Added solution.

### PR DESCRIPTION
**Pull request by Merkald**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
-        return null;
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName));) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                stringBuilder.append(line.toLowerCase().replaceAll("\\p{Punct}", ""));
+                stringBuilder.append(" ");
+            }
+        } catch (IOException e) {
+            return new String[0];
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:29]: you should re-throw exception, don't ignore them

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -1,14 +1,46 @@
...
+            if (stringBuilder.charAt(0) == WORD_START_AT) {
+                result.append(stringBuilder.subSequence(0, stringBuilder.indexOf(" "))).append(" ");
+            }
+            stringBuilder.delete(0, stringBuilder.indexOf(" ") + 1);
+        }
+        if (result.length() != 0) {
+            String[] finalResult = result.toString().split(" ");
+            Arrays.sort(finalResult);
+            return finalResult;
+        } else {
```
[the remark this answers is not in the document]
Comment (Merkald) [reply src/main/java/core/basesyntax/FileWork.java:42]: the condition is not fulfilled without this check returns String[1]="" And tests are fails.

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -1,14 +1,47 @@
...
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName));) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                stringBuilder.append(line.toLowerCase().replaceAll(ALL_PUNCTUATION, ""));
+                stringBuilder.append(" ");
+            }
+        } catch (RuntimeException | IOException e) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:29]: do we need to catch any RuntimeException here?

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -1,14 +1,47 @@
...
+            throw new RuntimeException();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:30]: add some informative message to exception

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/FileWork.java:43]: Yes, try to use List<String> instead of StringBuilder here

**Pull request closed without merging**